src/utils/validation.ts

```diff
@@ -1,9 +1,26 @@
 export default {
   isSafeInput(input: string): boolean {
     const maliciousPatterns = /[[]{}()'"<>\n;]/;
-    if (maliciousPatterns.test(input)) {
-      return false;
-    }
-    return true;
+    return !(maliciousPatterns.test(input));
+  },
+
+  isNotEmptyInput(input: string): boolean {
+    const emptyPattern = /^$/;
+    return !(emptyPattern.test(input));
+  },
+
+  isValidUID(input: string): boolean {
```

Comment:
Depending on the particular pattern for UID we're using, this could potentially be a little more stringent, but I'm not even sure that's necessary.